src/tests/evil/meson.build

```diff
@@ -0,0 +1,24 @@
+evil_test_src = files(
+'evil_suite.c',
+'evil_suite.h',
+'evil_test_dlfcn.c',
+'evil_test_main.c',
+'evil_test_stdio.c',
+'evil_test_stdlib.c',
+'evil_test_unistd.c',
+)
+
+evil_test_exe = executable('evil_suite',
+  include_directories : config_dir,
+  sources : evil_test_src,
+  dependencies: [m, check_dep, eina, evil],
```

Comment:
this tests depend on eina?

Reply:
Yes, it does. `evil_suite.h` includes `efl_check.h`, which in turn includes `Eina.h`. But great catch nonetheless, it does seem strange.